templates/index.html

```diff
@@ -24,20 +24,25 @@
             </div>
             </a>
             <a href="{{ url_for('payments.payouts') }}">
-                <div class='white-dashboard'>
-                    <p><center>Payouts</center></p>
-                </div>
-                </a>
+            <div class='dashboard'>
+                <p><center>Payouts</center></p>
+            </div>
+            </a>
             <a href="{{ url_for('test_email') }}">
             <div class='dashboard'>
                 <p><center>Test Email</center></p>
             </div>
             </a>
             <a href="{{ url_for('test_ws') }}">
-                <div class='dashboard'>
-                    <p><center>Test Websocket</center></p>
-                </div>
-                </a>
+            <div class='dashboard'>
+                <p><center>Test Websocket</center></p>
+            </div>
+            </a>
+            <a href="{{ url_for('user_compliance') }}">
+            <div class='dashboard'>
+                <p><center>User Compliance</center></p>
```

Comment:
Lets keep the verbiage consistent within the app

```suggestion
                <p><center>User KYC</center></p>
```

Reply:
changed compliance to kyc